### PR TITLE
[Docs]add "How to support new backends"中文翻译

### PR DESCRIPTION
**Pull request by LJoson**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

翻译了How to support new backends英文文档

## Modification

1.”Before contributing the codes“该描述感觉过于直白，翻译后自作主张调整为在对 MMDeploy 添加新的后端引擎之前
2，在模型转换第三部分代码示例中给的是onnxruntime,但是前面给的示例是trt,在说明中有说到他们之间联系是要一致，
我把这里的 (which should be same as the name in `mmdeploy/utils/constants.py`).

    **Example**:

    ```python
    backend_config = dict(type='onnxruntime')
    ```

代码改为
 **Example**:

    ```python
    backend_config = dict(type='tensorrt')
    ```

建议英文文档也可以改一下，不易引起不必要错误理解。
## Checklist

1. docs/zh_cn/tutorials/how_to_support_new_backends.md

### CODE REVIEW

**Comment by RunningLeon** [comment]
@LJoson Hi, thanks for the contribution. 
@VVsssssk @hanrui1sensetime Could you guy review this PR?

File: docs/zh_cn/tutorials/how_to_support_new_backends.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+    from mmdeploy.utils import Backend
+    from ..base import BACKEND_WRAPPER, BaseWrapper
+
+    @BACKEND_WRAPPER.register_module(Backend.TENSORRT.value)
+    class TRTWrapper(BaseWrapper):
+    ```
+
+2. 封装类可以在函数 `__init__` 中初始化引擎以及在 `forward` 函数中进行推理。请注意，该 `__init__` 函数必须接受一个参数 `output_names` 并将其传递给基类以确定输出张量的顺序。其中 `forward` 输入和输出变量应表示tensors的名称和值的字典。
+
+3. 为了方便性能测试，该类应该定义一个“执行”函数，只调用后端引擎的推理接口。该 `forward` 函数应在预处理数据后调用“执行”函数。
```
Comment (hanrui1sensetime) [inline docs/zh_cn/tutorials/how_to_support_new_backends.md:176]: May use "execute" instead of "执行", or may be users will confused of "执行"函数。

Comment (LJoson) [reply docs/zh_cn/tutorials/how_to_support_new_backends.md]: 哦，确实已改

**Review by hanrui1sensetime — DISMISSED** [review]
LGTM.

**Review by VVsssssk — DISMISSED** [review]

**Review by hanrui1sensetime — APPROVED** [review]
LGTM.

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**